src/Symfony/Bundle/TwigBundle/Resources/views/base_js.html.twig

```diff
@@ -45,13 +45,14 @@
                     var tabNavigation = document.createElement('ul');
                     tabNavigation.className = 'tab-navigation';
 
+                    var selectedTabId = 'tab-' + i + '-0'; /* select the first tab by default */
                     for (var j = 0; j < tabs.length; j++) {
                         var tabId = 'tab-' + i + '-' + j;
                         var tabTitle = tabs[j].querySelector('.tab-title').innerHTML;
 
                         var tabNavigationItem = document.createElement('li');
                         tabNavigationItem.setAttribute('data-tab-id', tabId);
-                        if (j == 0) { addClass(tabNavigationItem, 'active'); }
+                        if (hasClass(tabs[j], 'tab-active')) { selectedTabId = tabId; }
```

Comment:
why not check if class `'active'` is set already?

Reply:
Your proposal would work, but I think the current one is simpler.

Reply:
introducing tab-active + active :thinking: why use 2 "active" classes?

Reply:
Because we use `active` to apply other styles.

Reply:
I've reviewed this again ... and I was completely wrong. The `.active` styles are not used how I thought they were, so we can totally reuse that class. @ro0NL thanks for helping me realize this!